packages/shared/routes/dashboard/Dashboard.svelte

```diff
@@ -137,6 +109,39 @@
         }
     }
 
+    /**
+     * Handles deep link request
+     */
+    const handleDeepLinkRequest = (data) => {
+        const _redirect = (tab) => {
+            deepLinkRequestActive.set(true)
+            if (get(dashboardRoute) !== tab) {
+                dashboardRoute.set(tab)
+            }
+        }
+        if (!$appSettings.deepLinking) {
+            _redirect(Tabs.Settings)
+            showSystemNotification({ type: 'info', message: locale('notifications.deepLinkingIsNotEnabled') })
+        } else {
+            if ($accounts && $accounts.length > 0) {
+                let addressPrefix = $accounts[0].depositAddress.split('1')[0]
+
+                const parsedData = parseDeepLink(addressPrefix, data)
+
+                if (parsedData && parsedData.context === 'wallet' && parsedData.operation === 'send') {
+                    _redirect(Tabs.Wallet)
+                    console.log(parsedData)
```

Comment:
It might be a good idea to remove the console.log.

Reply:
good catch! Its removed